pkgs/development/libraries/crc32c/default.nix

```diff
@@ -29,6 +29,7 @@ stdenv.mkDerivation rec {
     "-DCRC32C_BUILD_BENCHMARKS=0"
     "-DCRC32C_USE_GLOG=0"
     "-DBUILD_SHARED_LIBS=${if staticOnly then "0" else "1"}"
+    "-DCMAKE_SKIP_BUILD_RPATH=OFF" # for tests
```

Comment:
Why not set this only if `stdenv.isDarwin` :thinking: ?

Reply:
Sure, may as well